aten/src/ATen/cpu/vec256/functional.h

```diff
@@ -169,4 +205,30 @@ inline void map2(
   }
 }
 
+template <typename scalar_t, typename Op>
+inline void map3(
+    const Op& vec_fun,
+    scalar_t* output_data,
+    scalar_t* input_data1,
```

Comment:
Correct me if I am wrong. 
https://github.com/pytorch/pytorch/blob/302e5662050e2dbfd0bc49db050f7c19cc06d8f3/aten/src/ATen/cpu/vec256/vec256_base.h#L186
It seems loadu() takes const ptr. All input data could be const scalar_t*, if so, you don't have to remove const in the later kernel functions.

Reply:
update the function signature.